admin_manual/installation/examples/lets_encrypt/register.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
- I am not happy with renaming the script to ``register.sh`` 
There was a reason behind naming it ``your-domain-name.sh``
It is adviced to have different scripts for different certificates and not only one. Each (sub)domain name will be listed in the certificate issued. For security reasons you may want to have seperate certificates for different subdomains! And: what will you do when issuing a new and different certificate for another domain and this script name...?
- Proposing to revert back the name of the script including on all places used.

Reply:
Sounds fair enough. I changed it for readability. No harm in reverting it.